README.md

```diff
@@ -144,11 +138,6 @@ In our experience, this project takes no less than 200 person-hours to build, an
 
 ## Getting Started
 
-### Folder organization
-![Rover build folders](images/folders.png)
-
-The main /osr/ folder contains all of the documentation and information necessary for the project, broken down into the 3 main sections: Mechanical, Electrical, and Software. Each of these sections is meant to be relatively self contained and should be fairly parallelizable, meaning that they could be completed simultaneously by different groups. There is a README.md in each section to help you navigate the information in that section. 
```

Comment:
i'm fine with getting rid of the folder organization image, but I think it's still useful to discuss the three main sections and that they're self contained and parallelizable. I'd leave some of this in

Reply:
That's mentioned in [the buildmap section](https://www.amazon.com/dp/B07DWVSR41/?coliid=ISMG8XMBILGFK&colid=1IM7RHK25RIBB&psc=0&ref_=lv_vv_lig_dp_it) as well. If that doesn't suffice, can you use the suggestion feature to add a blurb you think makes sense?